docker/Dockerfile

```diff
@@ -71,7 +72,11 @@ RUN TORCH_VER=$(curl --silent --location https://pypi.org/pypi/torch/json | pyth
     else \
         python -m pip install --no-cache-dir torch==$TORCH_VER+cpu torchvision==$TORCH_VISION_VER+cpu -f https://download.pytorch.org/whl/torch_stable.html; \
     fi
-RUN python -m pip install -U setuptools && python -m pip install --no-cache-dir captum torchtext torchserve torch-model-archiver
+RUN python -m pip install --no-cache-dir captum torchtext
+
+COPY . tmp/serve
+
+RUN python -m pip install --no-cache-dir tmp/serve/ tmp/serve/model-archiver
```

Comment:
So in our `Dockerfile.dev` we instead do this https://github.com/pytorch/serve/blob/master/docker/Dockerfile.dev#L61-L63

Any thoughts on what's better?

Reply:
torchserve & model-archiver & workflow-archiver are install at following line.
https://github.com/pytorch/serve/blob/0db95ee361d7f7a758a9e39f7cbc71e8c9340a4a/docker/Dockerfile.dev#L68

https://github.com/pytorch/serve/blob/master/ts_scripts/install_from_src.py

It seems simpler to call pip directly instead of 'install_from_src.py'.
From my observation, the installation step from the source seems to override the existing package. So uninstall in `install_from_src.py` also seems unnecessary.

In my personal opinion, it seems convenient to build a different version with the latest version of `Dockerfile.dev`, but I think it's reasonable to build only with the code from the (old) branch. If there is a serious error, it seems necessary to update the `Dockerfile` by hotfix.